cairo/src/error.rs

```diff
@@ -259,20 +269,51 @@ impl From<ffi::cairo_status_t> for Error {
     }
 }
 
-#[derive(Error, Debug)]
+#[derive(Debug)]
 pub enum IoError {
-    #[error("Cairo error: {0}")]
-    Cairo(#[from] Error),
-    #[error("IO error: {0}")]
-    Io(#[from] io::Error),
+    Cairo(Error),
+    Io(io::Error),
```

Comment:
You're missing the `Error::source()` / `cause()` implementations. @GuillaumeGomez added those it seems